WordPress/Classes/Models/ReaderSiteTopic.swift

```diff
@@ -49,7 +54,7 @@ import Foundation
         isJetpack = remoteInfo.isJetpack
         isPrivate = remoteInfo.isPrivate
         isVisible = remoteInfo.isVisible
-        isWPForTeams = remoteInfo.isWPForTeams
+        organizationID = remoteInfo.organizationID
```

Comment:
If I navigate to the discover feed (and pull to refresh if needed) organization_id is nil and an exception is thrown . Maybe `remoteInfo.organizationID ?? 0` to handle that?

Reply:
Not quite sure why the nil is happening there.  I didn't dig into it